packages/twenty-front/src/modules/object-record/record-index/components/RecordIndexRecordChip.tsx

```diff
@@ -17,6 +20,7 @@ export const RecordIdentifierChip = ({
   variant,
 }: RecordIdentifierChipProps) => {
   const { onIndexIdentifierClick } = useContext(RecordIndexRootPropsContext);
+  const recordIndexViewType = useRecoilValue(recordIndexViewTypeState);
```

Comment:
It feels like it would be cleaner to pass introduce size in this component and pass the chip size from the level right above, similar what we probably do with `variant` no? Can't tell exactly why but it feels a bit odd here